Footer Web and Hero Section

Reshex opened the pull request:
Changed footer component and scss, also added a new component for the hero section and added scss.
I haven't separated the pull request because i was waiting for a reply regarding the background and the font, and didn't want to wait in order to continue to the next task.
In the future ill make a separated branches for each section or feature.

Reshex committed:
changed footer

Reshex committed:
footer scss change and hero section component added

Reshex committed:
added lang change

Reshex committed:
fixes

Reshex committed:
added scss to hero section

Reshex committed:
changed background color

Reshex committed:
arrow down

vladbyk89 commented on src/components/Pages/HomeSections/footer/footerStyle.scss:
```diff
@@ -1,11 +1,12 @@
     align-items: center;
     position: relative;
-    background-color: rgb(117, 169, 228);
+    background-color: $BackgroundColor;
```
General colors that will be later reused in other CSS files should be defined in globals.scss

talyaron reviewed (changes requested):
Hi Bar,

Please look at the footer in mobile mode, and you will be able to see that it is not mobile frendly. please consult with Mash.

talyaron reviewed (changes requested):
Also, the Hero Section is not responsive. 
and please look at the image:
 <Image style={{ width: "38rem", height: "35rem" }} className="heroSectionImg" src={heroSectionImg} alt="Hero Section" />
the constant style should be only on the scss.

Thanks

Reshex committed:
adding footer responiveness

Reshex committed:
media query

Reshex committed:
fixed responsivness and added banner

talyaron reviewed (changes requested):
Hi bar, good work.
I need two changes:
- make the scss a module (css module). 
- about the varibales, please put them on the global.scss, with this method: --variable...

Reshex committed:
added css module, added variables(except one that wasnt inside the globals.scss) , added running banner responsivness, refactored and added modularity and fixed some bugs

vladbyk89 commented:
Good job so far. I have a few comments.
1) Make sure the text aligns the way it is presented in figma.
2) The bottom part, where the arrow is, could be shorter. (compare with figma design)

Reshex committed:
Fixed text align, sizes, and responsivness for the footer and the hero section

Reshex commented:
> Good job so far. I have a few comments.
> 
> 1. Make sure the text aligns the way it is presented in figma.
> 2. The bottom part, where the arrow is, could be shorter. (compare with figma design)

Hey Vladi, thank you for the feedback, I believe that you were referring only to the footer, I fixed the problems and also fixed some problems in the hero section component.
As i said before, I accidently made a mistake that both of the components are at the same branch, for the next task I will use another branch.

talyaron reviewed (approved):
Well done

Reshex committed:
Merge branch 'main' into footer-web
